app/src/androidTest/java/edu/uc/kovaciad/slicetracker/SQLInitTest.kt

```diff
@@ -22,7 +22,7 @@ import java.sql.SQLException
 @RunWith(AndroidJUnit4::class)
 class SQLInitTest {
 //
-//    private lateinit var db: SliceDatabase
+//    private lateinit val db: SliceDatabase
```

Comment:
Changed to val as database creation should be preserved according to databasebuilder function reference: https://developer.android.com/reference/android/arch/persistence/room/Room#databasebuilder

Reply:
Since this is commented out, I'd just remove it entirely.  :)

 Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.